lib/build.gradle.kts

```diff
@@ -0,0 +1,100 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("org.jetbrains.dokka")
+    id("maven-publish")
+}
+
+android {
+    namespace = "at.bitfire.vcard4android"
+
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 21        // Android 5
+        targetSdk = 33     // Android 13
+
+        aarMetadata {
+            minCompileSdk = 29
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+
+        isCoreLibraryDesugaringEnabled = true
+    }
+    kotlin {
+        jvmToolchain(17)
+    }
+
+    buildFeatures {
+        buildConfig = true
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+        }
+    }
+
+    packagingOptions {
+        resources {
+            excludes += listOf("LICENSE", "META-INF/LICENSE.txt", "META-INF/NOTICE.txt")
+        }
+    }
+
+    lint {
+        disable += listOf("AllowBackup", "InvalidPackage")
+    }
+
+    defaultConfig {
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    publishing {
+        // Configure publish variant
+        singleVariant("release") {
+            withSourcesJar()
+        }
+    }
+}
+
+publishing {
+    // Configure publishing data
+    publications {
+        register("release", MavenPublication::class.java) {
+            groupId = "com.github.bitfireAT"
+            artifactId = "vcard4android"
+            version = project.properties["vcard4android.version"] as String
+
+            afterEvaluate {
+                from(components["release"])
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.8.21")
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.3")
+
+    implementation("androidx.annotation:annotation:1.6.0")
+    // noinspection GradleDependency
```

Comment:
// noinspection seems to be Groovy syntax; Android Studio shows a warning that there's a newer version of this lib